.github/workflows/test.yml

```diff
@@ -86,35 +92,35 @@ jobs:
 
           - test-name: 'hs/tier1 common'
             test-suite: 'test/hotspot/jtreg/:tier1_common'
-            debug-suffix: -debug
+            debug-suffix: ${{ inputs.debug-suffix }}
```

Comment:
I don't understand why these changes were needed? What are you doing with the debug suffix?

Reply:
The hard-coded `debug-suffix: -debug` maps to run the hs/tier1 tests on `debug` build only. However, we can't build `debug` binary for static JDK yet due to the resource/space issue in GHA. I documented some related details in https://bugs.openjdk.org/browse/JDK-8355452?focusedId=14775969&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-14775969.

Changing hard-coded `-debug` to `${{ inputs.debug-suffix }}` allows us to set '' for `debug-suffix`, which maps to run hs/tier1 tests on `static-jdk` `release` build for now.

Reply:
Ok, I see. Then it makes sense.